bowtie/_cli.py

```diff
@@ -1096,6 +1107,19 @@ def _do_nothing(*args: Any, **kwargs: Any) -> CaseTransform:
 POSSIBLE_DIALECT_SHORTNAMES = ", ".join(sorted(Dialect.by_alias()))
 
 
+def pretty_names_str_for(dialects: Iterable[Dialect]) -> str:
```

Comment:
This is a nice kind of UX function -- but we can use [`inflect`](https://pypi.org/project/inflect/) for this, it has a `.join` function which does this.

Reply:
(And your special case will just be `== Dialect.known()` more or less at that point.)